app/views/users/index.html.erb

```diff
@@ -14,7 +14,7 @@
     <div class="col-md-12">
       <%= search_form_for(@search, class: "navbar-form navbar-left", url: users_path) do |f| %>
         <div class="form-group">
-          <%= f.search_field :user_username_or_user_email_or_member_uid_search_contains, class: "form-control" %>
+          <%= f.search_field :user_username_or_user_email_or_member_tags_or_member_uid_search_contains, class: "form-control" %>
```

Comment:
not sure how we can improve this :thinking:, but this name starts to be too much... maybe Ransack allows some kind "alias"?

Reply:
found it! https://github.com/activerecord-hackery/ransack#ransack-aliases

we can create a new one for both members lists 👌🏼  so we don't need to write and maintain this large expression everywhere :)